Source/WebCore/html/canvas/WebGLMultiDraw.cpp

```diff
@@ -187,6 +187,11 @@ bool WebGLMultiDraw::validateOffset(const char* functionName, const char* outOfB
         return false;
     }
 
+    if (offset >= static_cast<GCGLuint>(size)) {
+        m_context->synthesizeGLError(GraphicsContextGL::INVALID_OPERATION, functionName, outOfBoundsDescription);
+        return false;
+    }
+
     if (offset > static_cast<GCGLuint>(size - drawcount)) {
```

Comment:
Is there a bug or a test case for this?
Is there value of `offset` where the added the added hunk would return where the existing if below wouldn't?

`offset >= size`  is not the equivalent of  `offset > size - drawcount`
  if and only if `drawcount  == 0`

it should be valid to call:
offset = 5, size = 5, drawcount = 0  

The added if statement would make this invalid?